src/Tools/Performance Toolkit/App/src/BCPTLine.Codeunit.al

```diff
@@ -164,13 +164,14 @@ codeunit 149005 "BCPT Line"
     var
         BCPTLogEntry: Record "BCPT Log Entry";
         BCPTRoleWrapperImpl: Codeunit "BCPT Role Wrapper"; // single instance
-        IsHandled: Boolean;
         ValuesAreChanged: Boolean;
         BCPTTestSuite: Codeunit "BCPT Test Suite";
+        ModifiedMessage, ModifiedOperation : Text;
```

Comment:
You should add fields to the BCPT Log Entry table instead and use those.

Reply:
Of course... 🤦‍♂️